packages/@aws-cdk/aws-logs/lib/log-group.ts

```diff
@@ -316,6 +316,23 @@ export class LogGroup extends LogGroupBase {
     return new Import(scope, id);
   }
 
+  /**
+   * Import an existing LogGroup given its name
+   */
+  public static fromLogGroupName(scope: Construct, id: string, logGroupName: string): ILogGroup {
+    class Import extends LogGroupBase {
```

Comment:
can we reuse `fromLogGroupArn`?

Reply:
Why is that? Can you elaborate?

Reply:
Just less duplication (not that there is a lot, but still).

Reply:
```ts
public static fromLogGroupName(scope: Construct, id: string, logGroupName: string): ILogGroup {
  return this.fromLogGroupArn(scope, id, Stack.of(scope).formatArn({
    service: 'logs', resource: 'log-group', sep: ':', resourceName: logGroupName
  });
}
```

Reply:
Not a fan of this actually. Doing this makes the ARN a `Fn::Join` token and the loggroup name becomes a further token using more CF intrinsics. While all of this is technically valid, it seems to make things a bit complex.

I'd rather leave this as is.